lib/aruba/api.rb

```diff
@@ -206,6 +206,46 @@ def touch(*args)
       self
     end
 
+    # Copy a file and/or directory
+    #
+    # @param [String, Array] source
+    #   A single file or directory, multiple files or directories or multiple
+    #   files and directories. If multiple sources are given the destination
+    #   needs to be a directory
+    #
+    # @param [String] destination
+    #   A file or directory name. If multiple sources are given the destination
+    #   needs to be a directory
+    #
+    # rubocop:disable Metrics/CyclomaticComplexity
+    # rubocop:disable Metrics/PerceivedComplexity
+    def copy(*source, destination)
```

Comment:
I have another suggestion on the API, which is that we use an API more like Capybara's:

`copy 'foo', 'bar', to: 'baz/'

WDYT?